android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
+        <domain includeSubdomains="true">localhost</domain>
+    </domain-config>
```

Comment:
Just so I understand, this is allowing cleartext traffic but only from localhost and 10.0.2.2, right?

Reply:
Yes, correct. The default for Android emulators is to not allow cleartext traffic at all. The domain-config is only for the specified domains.